lib/internal/process/execution.js

```diff
@@ -60,35 +60,39 @@ function evalScript(name, body, breakFirstLine, print) {
   module.filename = path.join(cwd, name);
   module.paths = CJSModule._nodeModulePaths(cwd);
 
+  const { handleMainPromise } = require('internal/modules/run_main');
   const asyncESM = require('internal/process/esm_loader');
   const baseUrl = pathToFileURL(module.filename).href;
+  const { loadESM } = asyncESM;
+
+  handleMainPromise(loadESM(() => {
```

Comment:
Why not just split out this function to be called only when `--import` is passed, and then we don’t need to bother benchmarking it? Because the base case (no `--import` in use) wouldn’t meaningfully change.

Reply:
done